src/components/base/Toggle/index.jsx

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+import useToggle from '@hooks/useToggle';
+import { COLORS } from '@utils/constants';
+import PropTypes from 'prop-types';
+
+const ToggleContainer = styled.label`
+  display: inline-block;
+  cursor: pointer;
+  user-select: none;
+`;
+
+const ToggleSwitch = styled.div`
+  width: 54px;
+  height: 18px;
+  padding: 2px;
+  border-radius: 15px;
+  background-color: ${COLORS.grey};
+  transition: background-color 0.2 ease-out;
+  box-sizing: border-box;
+
+  &:after {
+    content: '';
+    position: relative;
+    top: -6px;
+    left: 0;
+    display: block;
+    width: 26px;
+    height: 26px;
+    border-radius: 50%;
+    background-color: ${COLORS.grey_10};
```

Comment:
```suggestion
    background-color: ${COLORS.grey_10};
    box-shadow: 0 1px 2px rgba(0, 0, 0, 0.2);
```
원본에 그림자가 쬐끔 있는 것 같은데 한번 넣어봐도 괜찮을 것 같아요!